wiki/Tournaments/SS_Tournament/2018/en.md

```diff
@@ -0,0 +1,205 @@
+1. The team that lost the roll is the team that picks first and it it alternates from there. 
+Example: BLUE rolls 67 RED rolls 10 the order would be 
+Blue BAN Red BAN  Blue BAN Red BAN : Red MAP Blue MAP Red MAP Blue MAP …
+1. Teams have 60 seconds to pick a map and 120 seconds to ready up refs will use '!mp timer' to keep time if a team doesn't pick in the first 60 seconds the opposing team will pick instead if the players are not ready after 120 seconds the ref will at their discretion start the match *Refs are in full control of any leniency here and may choose not to enforce this until semifinals at which point enforcement will be mandatory* 
+1. Once a map has been picked and everyone is ready the map will start after 5 seconds using '!mp start 5'
+1. If a player disconnects within the first 15 seconds of a map the referee will abort the match and the map will be restarted. Anyone found abusing this will be disqualified from the tournament. This may only happen once per team per match after the first dc the referee is not to abort
+
+
+## Mappools
+
+### Qualifiers
+
+![Qualifiers Mappool](Qaulifiers.jpg)
+
+- NoMod
+  - [Ceui - COLORFUL DAYS!!](https://osu.ppy.sh/b/203315) [Insane]
+  - [Mastermind(xi+nora2r) - Dreadnought](https://osu.ppy.sh/b/813331) [Another]
+  - [Function Phantom - Euclid] (https://osu.ppy.sh/b/971086) [toybot's Insane]	
+- DoubleTime
+  - [Duca - Clover](https://osu.ppy.sh/b/157856) [Collab]	
+  - [Icon For Hire - Make A Move](https://osu.ppy.sh/b/1077369) [Hard]		
+- Hidden  
+  - [Dreamcatcher - Sleep-walking](https://osu.ppy.sh/b/1370703) [Insane]	
+  - [yuiko - petit bonheur](https://osu.ppy.sh/b/1327955) [Insane]	
+- Hardrock
+  - [Otokaze - Karen (Short Ver.)](https://osu.ppy.sh/b/154357) [Hanabi]		
+  - [ALiCE'S EMOTiON - Dark Flight Dreamer](https://osu.ppy.sh/b/763294) [Twaoi's Insane]			
+### Group Stage
+
+![Group Stage Mappool](Group.jpg)
+
+- NoMod
+  - [GARNiDELiA - ambiguous](https://osu.ppy.sh/b/680886) [Insane]
+  - [Mind Vortex - Arc](https://osu.ppy.sh/b/239037) [NSane]
+  - [sakuzyo - VALLISTA](https://osu.ppy.sh/b/127313) [Another]
+  - [B-ko(Cv:Touyama Nao) - Nisemono Chuuihou](https://osu.ppy.sh/b/1253351) [Insane]
+  - [Mediks - Doomsday](https://osu.ppy.sh/b/1290316) [Insane]
+  - [Dark PHOENiX - Taketori Hishou](https://osu.ppy.sh/b/86324) [Extra]
+- Double Time
+  - [Two Steps From Hell - Star Sky](https://osu.ppy.sh/b/1270965) [Unstoppable]
+  - [Petit Rabbit's - No Poi!(nenpulse bootleg remix)](https://osu.ppy.sh/b/937188) [YakiBoss's Hard]
+  - [Chata - Koi wo Shiyouyo](https://osu.ppy.sh/b/81200) [Insane]
+- Hidden
+  - [Kano - Yuudachi no Ribbon](https://osu.ppy.sh/b/1078420) [Dailycare's Insane]
+  - [Renard - Terminal](https://osu.ppy.sh/b/270363) [Ends' Insane]
+  - [Ne;on & Haiji - Haruka Tooki Gensoukyo - Dream of fantasy](https://osu.ppy.sh/b/106475) [lkp's Lunatic]
+- Hardrock
+  - [Rchetype - V Is For] (https://osu.ppy.sh/b/290919) [Insane]
```

Comment:
ditto

Reply:
`[Rchetype - V Is For] (https://osu.ppy.sh/b/290919) [Insane]` ->
`[Rchetype - V Is For](https://osu.ppy.sh/b/290919) [Insane]`